Validation/RecoTrack/python/TrackingParticleSelectionsForEfficiency_cff.py

```diff
@@ -12,7 +12,7 @@
     ptMin = cms.double(0.9),
     ptMax = cms.double(1e100),
     maxRapidity = cms.double(2.5),
-    tip = cms.double(3.5),
+    tip = cms.double(2.0),
```

Comment:
is this something to be changed globally upstream, or only for a specific workflow ?

Reply:
I'd say globally, since this is within the Barrel Layer1 volume, while 3.5 was outside.

Reply:
Thanks, I've moved this to #339 .